api/src/test/java/io/opentelemetry/trace/TestUtils.java

```diff
@@ -30,9 +30,7 @@
    * @return a {@link TraceId}.
    */
   public static TraceId generateRandomTraceId(Random random) {
```

Comment:
This is still needed because it's used in other packages (`trace.samplers`). Alternative is to make `TraceId.generateRandomId` and `SpanId.generateRandomId` public.

Reply:
I would prefer to have this in tests, so we don't expose this public.

Reply:
I'm slightly leaning towards @bogdandrutu opinion - I think this is nice to have, but its best place, at least publicly, is the SDK.

Reply:
SG, these methods are package-private as for now.